Source/WebService/Blackjack.WebService/ClientApp/src/app/services/user.services.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from "@angular/core";
```

Comment:
Delete this file? Nothin is goin on in here. Looks like maybe http stuff was gonna be done in here but is bein done correctly in your user.api.ts file now. If this is not needed you don't wanna check in junk to the main branch